plugins/modules/net_tools/pritunl/pritunl_org.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Florian Dambrine <android.florian@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: pritunl_org
+author: "Florian Dambrine (@Lowess)"
+short_description: Manage Pritunl Organizations using the Pritunl API
+description:
+    - A module to manage Pritunl organizations using the Pritunl API.
+options:
+    pritunl_url:
+        type: str
+        required: true
+        description:
+            - URL and port of the pritunl server on which the API is enabled
+
+    pritunl_api_token:
+        type: str
+        required: true
+        description:
+            - API Token of a Pritunl admin user (It needs to be enabled in
+              Administrators > USERNAME > Enable Token Authentication).
+
+    pritunl_api_secret:
+        type: str
+        required: true
+        description:
+            - API Secret found in Administrators > USERNAME > API Secret.
+
+    state:
+        type: str
+        required: false
+        default: list
+        choices: [ list, present, absent ]
```

Comment:
`list` must be removed. Create a separate `pritunl_org_info` module for that. (See https://github.com/Akasurde/ansible/blob/1985d9f01ffcf1c4fe2d6d89cfeb6f1ee24fbe8b/docs/docsite/rst/dev_guide/developing_modules_general.rst#creating-an-info-or-a-facts-module for details.)